helpers/helpers.go

```diff
@@ -282,6 +283,28 @@ func RunCommandSilent(cmdname string, args ...string) error {
 	return err
 }
 
+// RunCommandTimeout runs the given command with timeout + args and does not print command output
+func RunCommandTimeout(timeout int, cmdname string, args ...string) error {
+	ctx := context.Background()
+	// 0 means infinite timeout, ONLY set timeouts when value is > 0
+	if timeout > 0 {
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithTimeout(context.Background(), time.Duration(timeout)*time.Second)
+		defer cancel()
+	}
+	cmd := exec.CommandContext(ctx, cmdname, args...)
+	cmd.Stdout = nil
```

Comment:
This part here looks like it could be replaced by RunCommandSilent, but alas, the function not use CommandContext.

Reply:
Yeah :( I don't like kind-of duplicate code but it's hard to merge them all.